### ISSUE TITLE
Pass arguments to publication

### ISSUE THREAD

**Issue by 57even** [aslagle/reactive-table#255]
Is it not possible to pass arguments to `ReactiveTable.publish`?

I have tried

    ReactiveTable.publish('commentsByPostId', function (postId) {
      check(postId, String);
      return Comments.find({ postId: postId });
    });

but the client cannot find the publication.

**Comment by aslagle** [comment]
No, it's not possible. You can create a custom filter for the postId field and set it to the postId you want.
